### PR TITLE
Updated channels example links and cargo

### PR DESCRIPTION
**Pull request by kwek20**
# Description of change

- Moved the channels-example link to mine instead of Jakes, as he wont be maintaining it any longer :(
- Updated iota.rs branch to `iota-1.0` as iota.rs now defaults to chrysalis-2.0

# Type of change
- Bug fix (a non-breaking change which fixes an issue)

# Change checklist

- [X] I have followed the contribution guidelines for this project
- [X I have performed a self-review of my changes

### CODE REVIEW

File: channels/1.3/getting-started/get-started-rust.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 **To get started with Channels, follow the suggested workflow in this topic.**
 
 :::info:
-The example code is also hosted on [GitHub](https://github.com/iotaledger/streams/tree/master/examples).
+The example code is also hosted on [GitHub](https://github.com/kwek20/channels-examples.git).
```
Comment (fijter) [inline channels/1.3/getting-started/get-started-rust.md:6]: I think the .git prefix is not needed, also don't we want these examples in the iotaledger repo instead?

**Comment by kwek20** [comment]
@fijter Well it used to be in iota-community, do we prefer it there? I dont have maintainer rights there though.
Didnt want to clutter our main github with it, but youre right htat it shouldnt be on a personal account :=)

**Comment by kwek20** [comment]
@fijter Talked with Mr. Cech and we moved it to iotaledger!

**Review by fijter — APPROVED** [review]
Sweet :+1:

**Pull request merged**